Makefile

```diff
@@ -37,12 +37,16 @@ _bootstrap-python:
 _bootstrap-githooks: clean-githooks
 	-(cd ${ROOTDIR}; scripts/dev/git-hooks.sh --install)
 
-# This should only be needed for CI rather than locally,
-# so we don't run it in `make bs`
-.PHONY: bootstrap_aws_creds
-bootstrap_aws_creds:
+# Each CI context can inject an AWS credential to the environment.
+# Unpack all context variables required and combine into a single credentials file.
+# Should only be needed for CI rather than locally.
+.PHONY: bootstrap-aws-context-creds-ci
+bootstrap-aws-context-creds-ci:
```

Comment:
Would be good to pull this whole logic into our existing orb patterns -- @gita-codaio should be able to give you directions on how to do that.

[coda/utils@2.0.0](https://circleci.com/developer/orbs/orb/coda/utils)